app/api/v1/sick_leaves.rb

```diff
@@ -42,6 +44,18 @@ def sick_leave
       route_param :id, type: Integer do
         get { sick_leave }
       end
+
+      desc 'Send pdf to patient'
+      route_param :id, type: Integer do
+        get :pdf do
+          patient = sick_leave.visits.first.patient
+          doctor = sick_leave.visits.first.user
```

Comment:
why do you take the first? I'd like to receive actual information about my 2nd, 3rd, etc visit

Reply:
Because we have one sick leave for all visits. Patient and doctor do not change from visit to visit